### DISCUSSION TITLE
Report icon is not there.

### DISCUSSION THREAD

**Discussion by psjafri** [JulianPrieber/littlelink-custom#833] [General]
I am just setting up LinkStack today. I tried to find out why the option to turn on the report icon under the administrator config seems to do nothing.
Is there any step that I missed when turning on the report icon on the user page?
I tried to find it in the wiki, but I cannot find any information regarding it.

Thank you in advance for your help and responses.

**Comment by JulianPrieber** [comment]
You need an admin email defined in your config.
Furthermore, report icons are not shown on VIP/verified or admin pages.
